app/controllers/orders_controller.rb

```diff
@@ -18,7 +17,29 @@ def show
     @order = Order.find(params[:id])
   end
 
-  def create
+  def new
+  end
 
+  def create
+    cart = Cart.new(session[:cart])
+    total_cost = 0
+    cart.items.each do |item, quantity|
+      total_cost += item.price * quantity.to_i
+    end
+    @order = Order.create(
+      user_id: session[:user_id],
+      total_cost: total_cost,
+      status: 'ordered'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.